capella_console_client/enumerations.py

```diff
@@ -89,3 +91,29 @@ class TaskingRequestStatus(str, BaseEnum):
     canceled = "canceled"
     error = "error"
     failed = "failed"
+
+
+class CollectionTier(str, BaseEnum):
+    high = "1_day"
```

Comment:
@log0s - can you please replace this with the new tier names (https://docs.capellaspace.com/constellation-tasking/tasking-requests#single-and-area-tasking)?

urgent, priority, standard, flexible, internal